src/lib/y2storage/disk_size.rb

```diff
@@ -221,6 +221,32 @@ def <=>(other)
       raise TypeError, "Unexpected #{other.class}; expected DiskSize"
     end
 
+    # Result of rounding up the size to the next value that is divisible by
+    # a given size. Returns the same value if it's already divisible.
+    #
+    # @param unit_size [DiskSize]
+    # @return [DiskSize]
+    def ceil(unit_size)
+      return DiskSize.new(@size) unless can_be_rounded?(unit_size)
+
+      unit_bytes = unit_size.to_i
+      modulo = @size % unit_bytes
+      bytes = modulo.zero? ? @size : @size + unit_bytes - modulo
+      DiskSize.new(bytes)
+    end
+
+    # Result of rounding down the size to the previous value that is divisible
+    # by a given size. Returns the same value if it's already divisible.
+    #
+    # @param unit_size [DiskSize]
+    # @return [DiskSize]
+    def floor(unit_size)
```

Comment:
Would it make sense to allow unit_size to be nil here? There's a lot of if(rounding) in the code below.